### ISSUE TITLE
[Question] Is KataGo 6b/10b...etc saturated?

### ISSUE THREAD

**Issue by Chenvincentkevin** [lightvector/KataGo#645]
Hi.

Excuse me if it was already discussed on Discord.
(It is hard to follow DIscord's thread...)

I found the Elo of the newest 6b is about 10k, and the highest is about 13.5k, but the 6b_network file is only about 5MB, while the Elo reached 13k the file enlarged to more than 100MB, and the Elo is just 1.3 times more.

**Comment by Chenvincentkevin** [comment]
So, are they saturated? And are there any possible ways that I can contribute to 6b's ratimg matches, or other versions of network's rating matches?

**Comment by lightvector** [comment]
The 6b is no longer being trained. It would be possible to improve it more by training it on recent data, but it will never be stronger than the current net given equal compute power invested, so even if it is not close to fully saturated, it is still saturated *enough* to not be worth running.

It sounds like you have a big misconception about Elo - "just 1.3 times more" is the wrong way to think about Elo. For example, suppose in some game you have two players, one with a 100000 Elo, and one with a 101000 Elo, and these ratings are accurate within the rating scale for that game. How much stronger is the second player? Are they only 1% stronger because their Elo is only 1% larger?

No. Under the Elo model, a difference of 1000 Elo corresponds to the second player winning 99.7% of the games. The second player is *crushingly stronger* than the first one.

Therefore, under the Elo model, it is meaningless to talk about "X times more Elo" or "X% larger Elo". What matters are the *absolute* differences. A difference of 1000 Elo is a enormous difference, no matter how much Elo there already is to begin with.

Another way to think about it is - the stronger you get, the more difficult and impressive it is to become even stronger. This is true for humans too, not just bots, even if it's on a different scale. This is why, for example, beginners improve can improve enormously after merely a handful of games, while mid-kyu players need to carefully practice over hundreds of games to improve and high dan or pro players can play thousands of games, and study intensely every day, and only improve barely. The amount of knowledge you have to accumulate per each point of Elo grows *exponentially*. 

So, if you like, let's make up a new rating system "alpha = 10^(Elo / 1500)". So a 1500 Elo player would be have 1 alpha, a 3000 Elo player would have 10 alpha, a 4500 Elo player would have 100 alpha, a 6000 Elo player would have 1000 alpha, and so on. (NOTE: The factor of 1500 would be highly dependent on the specific game, so don't blindly apply the same 1500 for Chess or other games).

You could consider "alpha" to be more closely proportional to something like "total learned knowledge" or "total skill experience" with a game. Unlike Elo, for alpha it *is* actually meaningful to talk about "X times more". The alpha of the current nets is roughly 200 times larger than the alpha of the best 60b net. Which is intuitively appropriate - the current 40b nets crushingly dominate the 6b nets, not only would they beat the 6b nets around 100% of the time, they could beat an opponent who could beat the 6b nets around 100% of the time, around 100% of the time.

**Comment by Chenvincentkevin** [comment]
Thanks for the detailed explanation, I understand it now. And wish all of us and the Katago program have further progress. As the saying goes the stronger you get, the more difficult and impressive it is to become even stronger. I want to my gratitude again since you pay a lot of time answering the question. The problem is solved.
